views/openweathermap-form-settings.twig

```diff
@@ -58,5 +58,9 @@
         {% set title %}{% trans "Cache Period" %}{% endset %}
         {% set helpText %}{% trans "This module uses 3rd party data. Please enter the number of seconds you would like to cache results." %}{% endset %}
         {{ forms.input("cachePeriod", title, interface.getSetting("cachePeriod", 3600), helpText) }}
+
+        {% set title %}{% trans "XMDS Cache Period" %}{% endset %}
+        {% set helpText %}{% trans "This module uses 3rd party data. Please enter the number of hours you would like to cache results from an XMDS call." %}{% endset %}
```

Comment:
If a player has weather based schedule criteria, how long should this connector cache that weather data for?

Reply:
Modified this prompt to include "hours" to let users know that the field is asking for the duration in hours.